analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/Foo/Web.config

```diff
@@ -0,0 +1,9 @@
+﻿<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Is it intentional, that this file have `pom.xml`-like structure? The intention of this test is not clear from `Foo` name.

Reply:
it is unexpected XML content, the intention should be in the test that references this. I agree that the naming can be improved